test/e2e/storage/csi_objects.go

```diff
@@ -140,6 +151,54 @@ func csiServiceAccount(
 	return ret
 }
 
+func csiPSPRoleBinding(client clientset.Interface,
+	config framework.VolumeTestConfig,
+	teardown bool,
+	sas []*v1.ServiceAccount) {
+	bindingString := "Binding"
+	if teardown {
+		bindingString = "Unbinding"
+	}
+	roleBindingClient := client.RbacV1beta1().RoleBindings(config.Namespace)
+	for _, sa := range sas {
```

Comment:
when do you need multiple service accounts?

Reply:
Recommended deployment of GCE PD uses two service accounts, one for node, the other for controller